it/src/test/java/com/sonarsource/scanner/it/SonarScannerTestSuite.java

```diff
@@ -38,23 +37,11 @@
   private static Orchestrator createOrchestrator() {
     String sonarVersion = System
       .getProperty("sonar.runtimeVersion", "LATEST_RELEASE[7.9]");
```

Comment:
Is this run only with LTS ? Should we add also latest DEV version ?

Reply:
No this is just the default. On Cirrus it will run against LTS and DEV